iothub/device/src/ModuleClient.cs

```diff
@@ -16,36 +18,40 @@
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// Contains methods that a device can use to send messages to and receive from the service.
+    /// Contains methods that a module can use to send messages to and receive from the service.
     /// </summary>
     public sealed class ModuleClient : IDisposable
     {
         private const string ModuleMethodUriFormat = "/twins/{0}/modules/{1}/methods?" + ClientApiVersionHelper.ApiVersionQueryStringLatest;
         private const string DeviceMethodUriFormat = "/twins/{0}/methods?" + ClientApiVersionHelper.ApiVersionQueryStringLatest;
-        private readonly InternalClient internalClient;
```

Comment:
is this getting set in places other than the constructor?

Reply:
not being set anywhere else, but used in tests.